<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IRTransUI</class>
 <widget class="QWidget" name="IRTransUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Icons</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>IP Adress</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="editIP">
     <property name="text">
      <string>192.168.x.x</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="editPort">
     <property name="text">
      <string>21000</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QPushButton" name="On">
     <property name="text">
      <string>On</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="3">
    <widget class="QLineEdit" name="On_Command"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="Off">
     <property name="text">
      <string>Off</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="QLineEdit" name="Off_Command"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBoxLog">
     <property name="text">
      <string>Log</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QTextEdit" name="logTxt"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
